Zpp/DbCache/DbTransactionData.cs

```diff
@@ -154,14 +155,12 @@ public void PersistDbCache()
                 _purchaseOrderParts.GetAllAs<T_PurchaseOrderPart>();
 
             // Insert all T_* entities
-
+            InsertOrUpdateRange(tProductionOrders, _productionDomainContext.ProductionOrders);
+            InsertOrUpdateRange(_productionOrderOperations, _productionDomainContext.ProductionOrderOperations);
             InsertOrUpdateRange(tProductionOrderBoms, _productionDomainContext.ProductionOrderBoms);
-            InsertOrUpdateRange(_productionOrderOperations,
-                _productionDomainContext.ProductionOrderOperations);
             InsertOrUpdateRange(tStockExchangeDemands, _productionDomainContext.StockExchanges);
             // providers
             InsertOrUpdateRange(tStockExchangesProviders, _productionDomainContext.StockExchanges);
-            InsertOrUpdateRange(tProductionOrders, _productionDomainContext.ProductionOrders);
```

Comment:
Could you please revert these changes (all InsertsOrUpdate..), or are they really needed? I just want to avoid side effects.

Reply:
I'd suggest to keep it like that. Because we changed it to reflect the required foreign key’s structure and way they are created.

Reply:
Ok.